state/txindex/kv/kv.go

```diff
@@ -426,6 +426,26 @@ func (txi *TxIndex) match(
 			}
 		}
 
+	case c.Op == query.OpExists:
+		// XXX: can't use startKeyBz here because c.Operand is nil
+		// (e.g. "account.owner/<nil>/" won't match w/ a single row)
+		it, err := dbm.IteratePrefix(txi.store, startKey(c.CompositeKey))
+		if err != nil {
+			panic(err)
+		}
+		defer it.Close()
+
+		for ; it.Valid(); it.Next() {
```

Comment:
This should check `it.Error()` after the loop ends, e.g.:

```go
if err := itr.Error(); err != nil {
  panic(err)
}
```

Reply:
what for?

Reply:
So that we don't silently ignore errors, which would lead to clients getting incomplete results?